<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogImageCapture</class>
 <widget class="QDialog" name="dialogImageCapture">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>281</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Capture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/logo.ico</normaloff>:/resources/logo.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelCameraStatus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>162</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCheckCamera">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>162</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCameraSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>162</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Settings</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textSaveLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>126</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowse">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>126</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaveLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>106</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>This tool allows the user to change camera settings and take images from the camera which are saved to the given location. Please check that a camera is available and browse to a save location before capturing images. Press Capture to take one picture. Press Run and run the 3D machine, where the magnetic trigger will take images as the machine completes the scan and coat processes.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDone">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>241</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCapture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>198</y>
     <width>161</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>198</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>228</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeElapsed">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>198</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Elapsed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>218</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatusBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonDone</sender>
   <signal>clicked()</signal>
   <receiver>dialogImageCapture</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
